release-notes/unreleased/IGAPP-955-sr-latn.yml

```diff
@@ -0,0 +1,5 @@
+issue_key: IGAPP-955
+show_in_stores: false
+platforms:
+  - web
+en: Fixed sr-Latn, sr-Cyrl and zh-CN languages.
```

Comment:
Not sure if this is understandable or if it is better to write the language as it is called ("Fixed serbian and chinese languages")

Reply:
Since this is just for us internally I think it is fine. Will change anyway.